shared/actions/favorite.js

```diff
@@ -164,6 +164,11 @@ const jsonToFolders = (json: Object, myKID: any) => {
 
 export function favoriteList (): (dispatch: Dispatch, getState: () => Object) => void {
   return (dispatch, getState) => {
+    // Ignore any messages we receive while reloading
```

Comment:
You still want some actions to happen. Like registering listeners.

So it's a bit tricky to know in the middleware which thunks you should allow to pass vs which ones you should ignore.